crypto/params.c

```diff
@@ -13,6 +13,16 @@
 #include "internal/thread_once.h"
 #include "internal/numbers.h"
 
+/*
+ * Return the number of bits in the mantissa of a double.  This is used to
+ * shift a larger integral value to determine if it will exactly fit into a
+ * double.
+ */
+static unsigned int real_shift(void)
```

Comment:
I know I am coming late, just wondering if it is worth to have a PR to make this explicilty `ossl_inline` or if it is superfluous and we can leave it to the compiler to optimize it without an hint.

@kroeckx @paulidale what do you think?

Reply:
I've not met a compiler this century that would fail to inline this.